spec/migrations/20200914005000_add_version_and_cloud_provider_to_configuration_profiles.rb

```diff
@@ -0,0 +1,20 @@
+require_migration
+
+describe AddVersionAndCloudProviderToConfigurationProfiles do
+  let(:config_profile_stub) { migration_stub(:ConfigurationProfile) }
+
+  migration_context :up do
+    it 'creates the new columns' do
+      config_profile = config_profile_stub.create!(:name => "test")
+
+      expect(config_profile).to_not respond_to(:default_version)
+      expect(config_profile).to_not respond_to(:cloud_provider)
```

Comment:
We don't commonly say you don't need tests :laughing:, but if you're just adding or removing columns not doing any data migrations you don't really need specs since you're just testing that ActiveRecord `add_column` works

Reply:
Good to know, thx :-)